src/app/beer_garden/config.py

```diff
@@ -179,8 +179,10 @@ def generate_plugin_logging(args: Sequence[str]) -> dict:
 
     Args:
         args: Command line arguments
-            --config-file: Configuration will be written to this file (will print to
-                stdout if missing)
+            --local_conf: Local plugin configuration will be written to this file
```

Comment:
This is related to the comment about just running the `generate_plugin_logging_config` command twice, but basically I think you don't necessarily need to change this file at all.